tests/catch/unit/memory/hipMalloc3DArray.cc

```diff
@@ -176,20 +126,144 @@ TEST_CASE("Unit_hipMalloc3DArray_MultiThread") {
   std::vector<std::thread> threadlist;
   int devCnt = 0;
   devCnt = HipTest::getDeviceCount();
-  size_t tot, avail, ptot, pavail;
-  HIP_CHECK(hipMemGetInfo(&pavail, &ptot));
+  const auto pavail = getFreeMem();
   for (int i = 0; i < devCnt; i++) {
-    threadlist.push_back(std::thread(Malloc3DArrayThreadFunc, i));
+    threadlist.push_back(std::thread(Malloc3DArray_DiffSizes, i));
```

Comment:
Add a TODO comment about the use of non-threadsafe ```HIP_CHECK```

Reply:
done